repository/assets_maintenance_repo.go

```diff
@@ -0,0 +1,22 @@
+package repository
+
+import (
+	"context"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/vkhichar/asset-management/domain"
+)
+
+type AssetMaintenanceRepo interface {
+	InsertMaintenanceActivity(ctx context.Context, assets_id domain.UUID, req domain.MaintenanceActivity) (*domain.MaintenanceActivity, error)
+}
+
+type assetMaintainRepo struct {
```

Comment:
For implementation struct like this, provide empty implementation of abstract methods from parent interface. Here, as you have not provided any implementation for function InsertMaintenanceActivity in assetMaintainRepo, the build fails for such scenario. Others working with this branch have to declare it themselves for successful build.
Make sure the build is passing before raising any PR.